tempest/run-tempest.sh

```diff
@@ -57,6 +59,16 @@ export NOSE_WITH_XUNIT=1
 export NOSE_XUNIT_FILE=tempest_xunit.xml
 
 cd /usr/share/openstack-tempest-juno/
+
+# Quick check using Javelin
+if javelin_check_resources; then
```

Comment:
what if you hit the case where you already ran javelin *before* the upgrade and these resources did not survive this upgrade, so it will return 1, you will go in this case and will it create the resources again, while we would detect that resources did not survive.